pkg/deploytest/deployment.go

```diff
@@ -103,45 +93,35 @@ func NewDeployment(conf *TestConfig) (*Deployment, error) {
 		logger = logging.Synchronize(logging.ConsoleDebugLogger)
 	}
 
-	// Create a simulated network transport to route messages between replicas.
-	fakeTransport := NewFakeTransport(conf.NumReplicas)
-
 	// Create a dummy static membership with replica IDs from 0 to len(replicas) - 1
-	membership := make([]t.NodeID, conf.NumReplicas)
-	for i := 0; i < len(membership); i++ {
-		membership[i] = t.NewNodeIDFromInt(i)
+	nodeIDs := make([]t.NodeID, conf.NumReplicas)
+	for i := 0; i < len(nodeIDs); i++ {
+		nodeIDs[i] = t.NewNodeIDFromInt(i)
+	}
+
+	// Create a simulated network transport to route messages between replicas.
+	var transportLayer LocalTransport
+	switch conf.Transport {
+	case "fake":
+		transportLayer = NewFakeTransport(nodeIDs)
```

Comment:
Why do we need "Layer"? Even `transport` name is so long for that purpose

Reply:
Because `transport` is used to identify a single instance of `net.Transport`. I renamed the `LocalTransport` interface to `LocalTransportLayer` to be more consistent.